bitnami/opensearch/values.yaml

```
@@ -3408,3 +3408,181 @@ dashboards:
+    ## E.g.
+    ## selector:
+    ##   matchLabels:
+    ##     app: my-app
+    ##
+    selector: {}
+    ## @param snapshots.persistence.size Size of data volume
+    ##
+    size: 8Gi
+    ## @param snapshots.persistence.storageClass Storage class of backing PVC
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    storageClass: ""
+    ## @param snapshots.persistence.subPath The subdirectory of the volume to mount to, useful in dev environments and one PV for multiple services
+    ##
+    subPath: ""
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param snapshots.podSecurityContext.enabled Enabled data pods' Security Context
+  ## @param snapshots.podSecurityContext.fsGroupChangePolicy Set filesystem group change policy
+  ## @param snapshots.podSecurityContext.sysctls Set kernel settings using the sysctl interface
+  ## @param snapshots.podSecurityContext.supplementalGroups Set filesystem extra groups
+  ## @param snapshots.podSecurityContext.fsGroup Set snapshots pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroupChangePolicy: Always
+    sysctls: []
+    supplementalGroups: []
+    fsGroup: 1001
+  ## @param snapshots.policies [default: SNAPSHOT_POLICY_API_BODY] Each object represents a snapshot policy in YAML form, which will be converted to JSON and then passed as the HTTP body data to the OpenSearch REST API.
+  ## NOTE: The field `snapshot_config.repository` in each policy must match a repo name in `snapshots.repositories`.
+  ##
+  policies:
+    default:
+      creation:
+        schedule:
+          cron:
+            expression: "0 20 * * *"
+            timezone: UTC
+      deletion:
+        condition:
+          max_age: 7d
+          min_count: 1
+        schedule:
+          cron:
+            expression: 0 20 * * *
+            timezone: UTC
+      description: Default snapshot policy
+      enabled: true
+      snapshot_config:
+        repository: default # This must match a repo name in `snapshots.repositories`
+  ## @param snapshots.repositories [default: SNAPSHOT_REPO_API_BODY] Each object represents a snapshot repository in YAML form, which will be converted to JSON and then passed as the HTTP body data to the OpenSearch REST API.
+  ## NOTE: The field `settings.location` in each repo must match the snapshot repo path configured in `snapshotRepoPath`, if its `type` is `fs`.
+  ##
+  repositories:
+    default:
+      settings:
+        location: /snapshots # This must match the snapshot repo path configured in `snapshotRepoPath`
```

```suggestion
        location: /snapshots
```

As you added a note above, I'd remove the comment here

---

Yeah, I agree, it feels a bit redundant. Let's remove it and also let's make sure the README generator works with my new parameters.